src/commons/error.rs

```diff
@@ -551,7 +600,9 @@ impl Error {
 
             Error::ApiInvalidCredentials(e) => ErrorResponse::new("api-invalid-credentials", &self).with_cause(e),
 
-            Error::ApiAuthError(e) => ErrorResponse::new("api-auth-error", &self).with_cause(e),
+            Error::ApiAuthPermanentError(e) => ErrorResponse::new("api-auth-error", &self).with_cause(e),
```

Comment:
These labels need adding to the Lagosta `en.json` translation file. Without them the hard-coded English message strings in the Krill code will be shown to the end user. I don't see a corresponding branch at https://github.com/NLnetLabs/lagosta/branches, we should create one for this work.